SimulationOutputReport/measure.rb

```diff
@@ -785,7 +785,11 @@ def get_outputs(timeseries_frequency,
 
     if include_timeseries_airflows
       @airflows.each do |airflow_type, airflow|
-        airflow.timeseries_output = get_report_variable_data_timeseries(['EMS'], airflow.ems_variables.map { |var| "#{var}_timeseries_outvar" }, UnitConversions.convert(1.0, 'm^3/s', 'cfm'), 0, timeseries_frequency)
+        shift = true
+        if airflow_type == AFT::ClothesDryerExhaust
+          shift = false
+        end
```

Comment:
You ended up removing this code (that _undoes_ the EMS shift for dryer exhaust), right?

I tested a few different timeseries outputs (1 min timestep w/ 1 min output, 1 min timestep w/ hourly output, 60 min timestep w/ hourly output) and all of the results match the logic you described above, which is that the exhaust should operate when the dryer does, but for a minimum of 24 minutes. And the dryer itself lags behind the washer by 1 hour. So everything looks really good here. 

At this point, I'm only confused by the fact that you thought at one point we had to undo the EMS shift for dryer exhaust. Were you just confused at the time? Are we in agreement that it is working correctly now?

Reply:
So when the EMS lag code was in there, it was shifting the clothes dryer exhaust up one hour (and therefore was occurring before the clothes dryer events). I excluded the shift for clothes dryer exhaust, so that it lined up with clothes dryer events. But then "Mechanical Ventilation" did not coincide with clothes dryer exhaust (because it was still be shifted up one hour). So then I just removed the shifting altogether.